src/components/sykmeldingpanel/sykmeldingperiode/SykmeldingPeriode.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { BodyShort, Detail, Heading } from '@navikt/ds-react';
+
+import { SykmeldingPeriodeFragment } from '../../../graphql/queries/react-query.generated';
+import CheckboxExplanation from '../../shared/checkboxexplanation/CheckboxExplanation';
+
+import { getArbeidsrelatertArsakText, getPeriodeDateRange, getPeriodeTitle } from './sykmeldigPeriodeUtils';
+import styles from './SykmeldingPeriode.module.css';
+
+interface Props {
+    periode: SykmeldingPeriodeFragment;
+}
+
+function SykmeldingPeriode({ periode }: Props): JSX.Element {
+    return (
+        <div className={styles.sykmeldingPeriodeRoot}>
+            <Heading size="xsmall" level="3">
+                {getPeriodeTitle(periode)}
+            </Heading>
+            <Detail size="small" spacing>
+                {getPeriodeDateRange(periode)}
+            </Detail>
+            <SykmeldingPeriodeDetail periode={periode} />
+        </div>
+    );
+}
+
+function SykmeldingPeriodeDetail({ periode }: Pick<Props, 'periode'>): JSX.Element | null {
+    switch (periode.__typename) {
+        case 'AktivitetIkkeMulig':
+            if (periode.arbeidsrelatertArsak) {
+                return (
+                    <div>
+                        <Detail>Forhold på arbeidsplassen vanskeliggjør arbeidsrelatert aktivitet</Detail>
+                        {periode.arbeidsrelatertArsak.arsak.map((arsak) => (
+                            <CheckboxExplanation key={arsak} text={getArbeidsrelatertArsakText(arsak)} />
+                        ))}
+                        {periode.arbeidsrelatertArsak.beskrivelse && (
+                            <div>
+                                <Detail>Nærmere beskrivelse:</Detail>
+                                <BodyShort size="small">{periode.arbeidsrelatertArsak.beskrivelse}</BodyShort>
+                            </div>
+                        )}
+                    </div>
+                );
+            }
+            return <div />;
+        case 'Avventende':
+            return (
+                <div>
+                    <Detail>Innspill til arbeidsgiver om tilrettelegging</Detail>
+                    <BodyShort size="small">{periode.tilrettelegging}</BodyShort>
+                </div>
+            );
+        case 'Gradert':
+            return (
+                <div>
+                    {periode.reisetilskudd && (
+                        <CheckboxExplanation text="Pasienten kan være i delvis arbeid ved bruk av reisetilskudd" />
+                    )}
+                </div>
+            );
+        case 'Behandlingsdager':
+            return null;
+        case 'Reisetilskudd':
+            return null;
```

Comment:
Kjem det noko for `Behandlingsdager` og `Reisetilskudd` her etter kvart, eller skal det ikkje visast?

Reply:
Vi trenger å gå litt inn i detaljene på dette med Bendik. Men jeg _tror_ ikke vi skal ha noen ekstra detaljer på disse liste-itemsene.

Men jeg tenker at vi tar en approach hvor vi får på plass det meste, også finpusser vi sammen med Bendik når ting er litt mer komplett. :smile: